MyCryptoHQ/MyCrypto
# Add custom token

Closes #2285

### Description

Fixes `Save` action of Add custom token form.

### Changes

* The access to `tokens` in configuration.

### Steps to Test

1. Go to Add Custom Token form, paste an ERC20 address and click "Save".
2. Check the new token is listed.
